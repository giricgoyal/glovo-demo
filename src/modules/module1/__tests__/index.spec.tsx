import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import Module1 from 'src/modules/module1'

describe('modules/module1', () => {
    let container
    const renderer = new ShallowRenderer()

    beforeEach(() => {
        container = renderer.render(<Module1 />)
    })

    test('should render and match snapshot', () => {
        expect(container).toMatchSnapshot()
    })
})
