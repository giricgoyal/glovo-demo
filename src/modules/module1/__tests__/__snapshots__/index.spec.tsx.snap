// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modules/module1 should render and match snapshot 1`] = `
<React.Fragment>
  Module1
</React.Fragment>
`;
